common/lib/xmodule/xmodule/js/spec_util.js

```diff
@@ -0,0 +1,21 @@
+(function () {
+    'use strict';
+
+    jasmine.waitForInputAjax = function (conditionalFn) {
```

Comment:
In order to make these available to other repos, I think these should go into the UI Toolkit. If you think they are not purely AJAX related, then they could live in the same directory as ajax-helpers, either in spec-helpers.js or in a new class.